rewrite-core/src/main/java/org/openrewrite/TypeReferenceProvider.java

```diff
@@ -15,13 +15,13 @@
  */
 package org.openrewrite;
 
-import org.openrewrite.trait.TypeReference;
+import org.openrewrite.trait.reference.Reference;
```

Comment:
Do we typically have subpackages for traits? Unless we add more types to this package, I think we can keep it all in the `trait` package.